packages/odata-service-writer/src/index.ts

```diff
@@ -71,12 +71,13 @@ async function generate(basePath: string, service: OdataService, fs?: Editor): P
     const paths = await findProjectFiles(basePath, fs);
     ensureExists(basePath, ['webapp/manifest.json'], fs);
     enhanceData(service);
-
+    // set isServiceTypeEdmx true if service is EDMX
+    const isServiceTypeEdmx = (service.type ?? ServiceType.EDMX) === ServiceType.EDMX;
```

Comment:
```suggestion
    const isServiceTypeEdmx = service.type === ServiceType.EDMX
```

Reply:
Thanks Iain - I moved to default setting to defaults.